src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java

```diff
@@ -229,6 +229,9 @@
         GOOGLE_TOKENS_IN_CONFIG_TO_IGNORE.put("RightCurly", Stream.of(
                 // Until https://github.com/checkstyle/checkstyle/issues/4178
                 "LAMBDA").collect(Collectors.toSet()));
+        GOOGLE_TOKENS_IN_CONFIG_TO_IGNORE.put("NoWhitespaceBefore", Stream.of(
+                // Disabled in default configuration.
```

Comment:
This is not a valid reason why google doesn't want these enabled. This is like saying we disable these tokens because we have them disabled.

Please update to similar reason we have in our own config: `google uses GenericWhitespace for this behavior`